src/object_properties.cpp

```diff
@@ -164,4 +166,5 @@ void ObjectProperties::deSerialize(std::istream &is)
 	breath_max = readU16(is);
 	eye_height = readF1000(is);
 	zoom_fov = readF1000(is);
+	use_texture_alpha = readU8(is);
```

Comment:
This will read garbage on older servers. Please check whether there's still data available or use try/catch.

Reply:
Surely this won't matter unless you want to backport it?

Reply:
Indeed we break the backwards compatibility in this version. Never mind, the older clients will hopefully handle this without a crash.